dumb-init.c

```diff
@@ -136,15 +149,21 @@ void print_help(char *argv[]) {
 }
 
 
+void write_sigterm_replacement(char *arg) {
+    sigterm_replacement = strtol(arg, NULL, 10);
+}
```

Comment:
I think this function is probably unnecessary and the line could just be put in the `switch`